src/clr-angular/forms/common/utils/constants.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+/**
+ * This is the maximum number of column cells of the layout.
+ */
+export const COLUMN_SIZE: number = 12;
```

Comment:
I think we can just store this as a `readonly` property of the layout service to avoid this extra const and file.

Reply:
This would imply injecting the layout service into the control class service in order to calculate the appropiate col size for any given container

Reply:
I don't mind injecting the service into another service, we do this a lot and seems easier than standalone `const` values.